src/main/kotlin/services/processor/MessageProcessorService.kt

```diff
@@ -0,0 +1,130 @@
+package es.wokis.services.processor
+
+import dev.kord.common.entity.Snowflake
+import dev.kord.core.entity.Message
+import dev.kord.core.entity.ReactionEmoji
+import es.wokis.dispatchers.AppDispatchers
+import es.wokis.utils.asRegex
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.launch
+import java.util.logging.Level
+import java.util.logging.Logger
+import kotlin.coroutines.CoroutineContext
+
+private const val FIXED_UP_TWITTER_URL = "https://fixupx.com"
+private const val FIXED_UP_INSTAGRAM_URL = "https://ddinstagram.com"
+private const val FIXED_UP_REDDIT_URL = "https://rxddit.com"
+private const val FIXED_UP_TIKTOK_URL = "https://tnktok.com"
+
+private const val ISAAC_ID = 378213328570417154
+private const val FRAN_ID = 651163679814844467
+private const val GUTI_ID = 899918332965298176
+
+private const val FRANCE = "francia"
+private const val SPAIN = "españa"
+private const val MEXICO = "mexico"
+
+class MessageProcessorService(
+    private val appDispatchers: AppDispatchers
+) {
+    private val coroutineScope = CoroutineScope(SupervisorJob() + appDispatchers.io + coroutineExceptionHandler())
+    private val twitterLinks = listOf(
```

Comment:
Similar to Instagram, we can improve proper detection if we process only urls with the path /status/